napari/_app_model/actions/_type.py

```diff
@@ -0,0 +1,85 @@
+import functools
+import inspect
+from typing import Callable
+
+from app_model.types import Action
+from pydantic import Field
+
+from napari.utils.translations import trans
+
+
+class RepeatableAction(Action):
+    repeatable: bool = Field(
```

Comment:
Looking at how we are using it (isinstance check), I think the boolean flag here is not necessary, we would only initialize the action to be a repeatable one if this is true, and this flag is duplicate in that sense